src/app/shared/cookie-popup/cookie-popup.html

```diff
@@ -5,9 +5,9 @@
     <a
       href="https://policies.google.com/technologies/cookies"
       mat-button
-      color="accent"
+      color="primary"
       target="_blank"
       rel="noopener">More details</a>
-    <button mat-button color="accent" (click)="accept()">Ok, Got it</button>
+    <button mat-button color="primary" (click)="accept()">Ok, Got it</button>
```

Comment:
I'm not sure whether the spec allows more than one primary button next to each other.

Reply:
I'm not familiar with that guidance. Do you have a link to that part of the spec?

Reply:
I couldn't dig anything up from the spec, but my thinking was that you only have one primary action in a component. In this case the main action is accepting cookies which is more important compared to reading about cookies, because people likely know about them already.

Reply:
Got it. We'll try removing the coloring from the "More Details" button.